tests/web3js/eth_get_filter_logs_test.js

```diff
@@ -0,0 +1,92 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const conf = require('./config')
+const helpers = require('./helpers')
+const web3 = conf.web3
+chai.use(chaiHttp);
+
+it('returns a null result for missing filter', async () => {
+  // check that null is returned when the filter could not be found.
+  let response = await callRPCMethod('eth_getFilterLogs', ['0xffa1'])
+
+  chai.assert.equal(200, response.status)
+  chai.assert.isUndefined(response.body['result'])
+})
+
+it('create logs filter and call eth_getFilterLogs', async () => {
+  // deploy contract
+  let deployed = await helpers.deployContract('storage')
+  let contractAddress = deployed.receipt.contractAddress
+
+  // create logs filter on the address of the deployed contract
+  let response = await callRPCMethod('eth_newFilter', [{ 'address': contractAddress }])
```

Comment:
why we don't use the web3.js client for this?

Reply:
Unfortunately [web3.eth](https://web3js.readthedocs.io/en/v1.2.11/web3-eth.html) does not expose any methods for working with filters. So the only way around was to simply create a POST HTTP call.